### PR TITLE
Added Borrowed Books Tab

### PR DESCRIPTION
**Pull request by Sachinbisht27**
Targets #2337 

### Description
Added borrowed books tab in knowledge cafe.

### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: resources/views/knowledgecafe/library/menu.blade.php [hunk 1]
```diff
@@ -16,8 +22,14 @@
...
+         $params = array_merge($request,  ['wishlist' => 'booksInWishlist']);
+        @endphp
+        <a class="nav-item nav-link  {{ (request()->input('wishlist','active') === 'booksInWishlist') ? 'active' : '' }}"  href="{{ route('books.index', $params)  }}"><i class="fa fa-book"></i>&nbsp;Wishlist Books</a>
+    </li>
+    <li class="nav-item">
+        @php
+          $params = array_merge($request,  ['borrowedBook' => 'markedAsBorrowed']);
         @endphp
-        <a class="nav-item nav-link {{ (request()->input('wishlist','active') === 'booksInWishlist') ? 'active' : '' }}"  href="{{ route('books.index', $request)  }}"><i class="fa fa-book"></i>&nbsp;Wishlist Books</a>
+        <a class="nav-item nav-link {{ (request()->input('borrowedBook','active') === 'markedAsBorrowed') ? 'active' : '' }}"  href="{{ route('books.index', $params)  }}"><i class="fa fa-book"></i>&nbsp;Borrowed Books</a>
```
Comment (gautam071) [inline resources/views/knowledgecafe/library/menu.blade.php:33]: Did you replace the wishlist to borrowed?

**Review by gautam071 — CHANGES_REQUESTED** [review]

File: resources/views/knowledgecafe/library/menu.blade.php (hunk 1 above)
Comment (Sachinbisht27) [reply resources/views/knowledgecafe/library/menu.blade.php:33]: I replaced wishlist with borrowed but, before that i added wishlist tab code above. (line 27)

File: resources/views/knowledgecafe/library/menu.blade.php [hunk 2]
```diff
@@ -16,8 +22,14 @@
     </li>
     <li class="nav-item">
         @php
-         $request['wishlist'] = 'booksInWishlist';
+         $params = array_merge($request,  ['wishlist' => 'booksInWishlist']);
+        @endphp
+        <a class="nav-item nav-link  {{ (request()->input('wishlist','active') === 'booksInWishlist') ? 'active' : '' }}"  href="{{ route('books.index', $params)  }}"><i class="fa fa-book"></i>&nbsp;Wishlist Books</a>
```
Comment (Sachinbisht27) [inline resources/views/knowledgecafe/library/menu.blade.php:27]: here

File: resources/views/knowledgecafe/library/menu.blade.php (hunk 1 above)
Comment (Sachinbisht27) [reply resources/views/knowledgecafe/library/menu.blade.php:33]: i tested both wishlist and borrowed working good

**Review by gautam071 — APPROVED** [review]
Looks good

**Pull request merged**